src/langsrv/langsrv.go

```diff
@@ -94,6 +96,19 @@ func writeMessage(message interface{ IMessage() }) error {
 	return err
 }
 
+func isFileScheme(documentURI uri.URI) bool {
+	u, err := url.ParseRequestURI(string(documentURI))
```

Comment:
It's unfortunate that `uri.URI` does not store scheme information after a `Parse`. :(

Reply:
Yes, it is a pain